code/modules/cargo/centcom_podlauncher.dm

```diff
@@ -589,49 +684,75 @@
 			if(0) //If we are launching all the turfs at once
 				for (var/turf/T in acceptableTurfs)
 					launchList |= typecache_filter_list_reverse(T.contents, ignored_atoms) //We filter any blacklisted atoms and add the rest to the launchList
+					if (iswallturf(T))
+						launchList += T
 			if(1) //If we are launching one at a time
 				if (launchCounter > acceptableTurfs.len) //Check if the launchCounter, which acts as an index, is too high. If it is, reset it to 1
 					launchCounter = 1 //Note that the launchCounter index is incremented in the launch() proc
-				for (var/atom/movable/O in acceptableTurfs[launchCounter].contents) //Go through the acceptableTurfs list based on the launchCounter index
-					launchList |= typecache_filter_list_reverse(acceptableTurfs[launchCounter].contents, ignored_atoms) //Filter the specicic turf chosen from acceptableTurfs, and add it to the launchList
+				var/turf/T = acceptableTurfs[launchCounter]
+				for (var/atom/movable/O in T.contents) //Go through the acceptableTurfs list based on the launchCounter index
+					launchList |= typecache_filter_list_reverse(T.contents, ignored_atoms) //Filter the specicic turf chosen from acceptableTurfs, and add it to the launchList
```

Comment:
What is going on here?
Also, please use better variable names than `O` and `T`

Reply:
`typecache_filter_list_reverse` iirc returns a list of everything in `T.contents` except for things that have a matching type in `ignored_atoms`

Reply:
wait nevermind haha what the fuck was I doing here